sections/basics/components-as-selectors.js

```diff
@@ -0,0 +1,57 @@
+import md from 'components/md'
+
+const ComponentSelectors = () => md`
```

Comment:
Don't we have docs about this somewhere already? I think in the FAQ?

The FAQ also has a better example than the one here, which shows how to target oneself which imo makes a lot more sense with styled-components than targeting other components.

Reply:
Oh yeah, you're right. I would not have thought to look for it in the FAQ... should we move the existing one to a more traffic'ed section?

Reply:
Yeah maybe, let's move it to the advanced section and use the FAQ entry there instead? I don't really feel like this is a basic thing imo.

Reply:
Agreed :) it it’d fit in better in “Advanced”